test/adaptiveCardPayloads/inputForm.ts

```diff
@@ -0,0 +1,100 @@
+export const inputForm = {
+  $schema: 'http://adaptivecards.io/schemas/adaptive-card.json',
```

Comment:
Do we even use all of these adaptive cards responses?

Reply:
Can you raise a ticket to follow up on the adaptive cards questions?

Reply:
Used in the dev panel.